src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -247,24 +247,27 @@ public async Task<Manifests> UpdateManifestsAutonomously(Manifests manifests)
 
                 DisplayMismatchedArchitectures(detectedArchOfInstallers);
             }
-            catch (InvalidOperationException)
+            catch (Exception e)
             {
-                Logger.ErrorLocalized(nameof(Resources.InstallerCountMustMatch_Error));
-                return null;
-            }
-            catch (ParsePackageException parsePackageException)
-            {
-                parsePackageException.ParseFailedInstallerUrls.ForEach(i => Logger.ErrorLocalized(nameof(Resources.PackageParsing_Error), i));
-                return null;
-            }
-            catch (InstallerMatchException installerMatchException)
-            {
-                Console.WriteLine();
-                Logger.ErrorLocalized(nameof(Resources.NewInstallerUrlMustMatchExisting_Message));
-                installerMatchException.MultipleMatchedInstallers.ForEach(i => Logger.ErrorLocalized(nameof(Resources.UnmatchedInstaller_Error), i.Architecture, i.InstallerType, i.InstallerUrl));
-                installerMatchException.UnmatchedInstallers.ForEach(i => Logger.ErrorLocalized(nameof(Resources.MultipleMatchedInstaller_Error), i.Architecture, i.InstallerType, i.InstallerUrl));
-                Console.WriteLine();
-                Logger.WarnLocalized(nameof(Resources.ResolveMatchingConflicts_Message));
+                if (e is InvalidOperationException)
+                {
+                    Logger.ErrorLocalized(nameof(Resources.InstallerCountMustMatch_Error));
+                }
+                else if (e is IOException)

```

Comment:
Needs filter here too

Reply:
Oops, I guess I missed that. Added filter